index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/16d2250538.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <title>Apple</title>
+  </head>
+  <body>
+    <header>
+      <div class="nav-bar">
+        <button type="button" id="applebutton">
+          <i class="fab fa-apple"></i>
+        </button>
+        <button type="button" id="storebutton">Store</button>
+        <button type="button" id="macbutton">Mac</button>
+        <button type="button" id="ipodbutton">iPod</button>
+        <button type="button" id="iphonebutton">iPhone</button>
+        <button type="button" id="ipadbutton">iPad</button>
+        <button type="button" id="itunesbutton">iTunes</button>
+        <button type="button" id="supportbutton">Support</button>
+        <input type="text" id="searchbutton" /><i class="fas fa-search"></i>
```

Comment:
- [ ] [Required] your search icon is flying outside of the input, please fix it ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

![image](https://user-images.githubusercontent.com/11758151/90667216-62b77600-e246-11ea-8ee1-d09f3885de78.png)